doc/radosgw/s3/bucketops.rst

```diff
@@ -59,7 +59,9 @@ HTTP Response
 ~~~~~~~~~~~~~
 
 If the bucket name is unique, within constraints and unused, the operation will succeed.
-If a bucket with the same name already exists and the user is the bucket owner, the operation will succeed.
+If a bucket with the same name already exists and the user is the
+bucket owner, the operation will succeed unless non-default option
+rgw_bucket_eexist_override is `true`.
```

Comment:
Double backticks around the option name please.  

And do you mean to have `eexist` with two `e`s ?

Reply:
1. ack
2. yes, it's from EEXIST